packages/twenty-server/src/engine/api/graphql/workspace-query-runner/factories/query-runner-args.factory.ts

```diff
@@ -39,9 +38,10 @@ export class QueryRunnerArgsFactory {
     const fieldMetadataMapByNameByName =
       options.objectMetadataItemWithFieldMaps.fieldsByName;
 
-    const shouldBackfillPosition = hasPositionField(
-      options.objectMetadataItemWithFieldMaps,
-    );
+    const shouldBackfillPosition =
+      options.objectMetadataItemWithFieldMaps.fields.some(
```

Comment:
Why removing the utility function, this one was only used there ?

Reply:
Yes